apps/react-storybook/stories/utils/wrappers.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useEffect } from "react";
+import * as inferno from "inferno";
+
+export function InfernoToReact<TProps>(component) {
```

Comment:
I guess lowerCamelCase is used for HOCs names in React:

```suggestion
export function asInfernoInReact<TProps>(component) {
```

or
```suggestion
export function wrapInfernoWithReact<TProps>(component) {
```